src/main/java/net/dv8tion/jda/internal/entities/EntityBuilder.java

```diff
@@ -817,12 +817,12 @@ public Message createMessage(JSONObject jsonObject, MessageChannel chan, boolean
                 JSONObject applicationData = jsonObject.getJSONObject("application");
 
                 final String name = applicationData.getString("name");
-                final String description = applicationData.getString("description");
-                final String iconId = applicationData.getString("icon");
-                final String coverId = applicationData.getString("cover_image");
+                final String description = applicationData.optString("description", applicationData.optString("summary", null));
+                final String iconId = applicationData.optString("icon", null);
+                final String splashId = applicationData.optString("splash", null);
```

Comment:
According to night, this is an internal field and `cover_image` is still correct. It just needs to be made optional (nullable).